helm_deploy/prisoner-content-hub-frontend/values-development.yaml

```diff
@@ -7,7 +7,7 @@ application:
     feedbackServiceName: aws-es-proxy-service
     feedbackEndpoint: /prod-feedback/_doc
     analyticsSiteId: UA-152065860-4
-    mockAuthEnabled: true
+    mockAuthEnabled: false
```

Comment:
is the plan to include mockAuthEnabled false in dev for now?

obviously you needed this to test but do you think we should have this as the default for a bit?

It seems like mockAuthEnabled: true is helpful when working on profile bits which we haven't done for a while

(I think we should fix the urls so we can support wildcards so it's less of an issue switching between the two modes - but outside of this ticket)

Reply:
Agreed, I think pushing the fix urls up the backlog would be a good idea